scripts/troubleshoot/LogCollection/AKSInsightsLogCollection.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation.
+#
+# This script will collect all logs from the replicaset agent pod and a random daemonset pod, also collect onboard logs with processes
+#
+# Author Nina Li
+
+Red='\033[0;31m'
+Cyan='\033[0;36m'
+NC='\033[0m' # No Color
+
+init()
+{
+    echo -e "Preparing for log collection..." | tee -a Tool.log
+
+    if ! cmd="$(type -p kubectl)" || [[ -z $cmd ]]; then
+        echo -e "${Red}Command kubectl not found, please install it firstly, exit...${NC}"
+        cd ..
+        rm -rf $output_path
+        exit
+    fi
+
+    if ! cmd="$(type -p tar)" || [[ -z $cmd ]]; then
+        echo -e "${Red}Command tar not found, please install it firstly, exit...${NC}"
+        cd ..
+        rm -rf $output_path
+        exit
+    fi
+
+    cmd=`kubectl get nodes 2>&1`
```

Comment:
there is no kubeconfig context passed. can you please add the kubeconfigContext as parameter to the script or the README updated to set the  context  of the cluster before running this script?

Reply:
updated README to set the context